src/screens/IndexScreen.js

```diff
@@ -195,6 +196,10 @@ export const IndexScreen = ({ navigation, route }) => {
         listItems = sortPOIsByDistanceFromPosition(listItems, position.coords);
       }
 
+      if (queryVariables?.genericType === GenericType.Deadline) {
+        listItems = _sortBy(listItems, (item) => item.title);
+      }
```

Comment:
I will remove this second commit, as we do not need it.
The data can be requested with correctly ordered items now:

```
"queryVariables": {
  "genericType": "Deadline",
  "limit": 15,
  "order": "title_ASC"
}
```